mets_mods2teiHeader/api/mets.py

```diff
@@ -280,6 +289,12 @@ def get_publishers(self):
         """
         return self.publishers
 
+    def get_edition(self):
```

Comment:
What's the advantage of `self.get_edition()` over `self.edition`?

I think we had that discussion before though but I forgot your reasoning.

Reply:
It's basically a left-over from C++ times. Obviously, getter/setter are not recommended (http://dirtsimple.org/2004/12/python-is-not-java.html) in Python. #21